### ISSUE TITLE
[Suggestion] Match emulation in options menu

### ISSUE THREAD

**Issue by Pichu0102** [synkarius/saltbot#90]
It would be nice if, in options, there was a way to emulate a match from two dropdowns and see the script output what it would do as if it were a scientist calculating for a real match and what it displays in the log during real matches.

**Comment by synkarius** [comment]
@reconman -- I agree. This is a cool idea.

**Comment by ACIIL** [comment]
Yes this would make testing so much easier, we live the stone ages of testing.   But i think the code to handle this is a ugly block doing everything for prediction, and not easy like inserting into the salty waifu bot match reader? I'll let the brave touch the code block before me.

**Comment by calexil** [comment]
bravery intensifies
